src/Propel/Generator/Reverse/OracleSchemaParser.php

```diff
@@ -228,6 +224,24 @@ protected function addIndexes(Table $table): void
         }
     }
 
+    /**
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return array<scalar, non-empty-list<null|scalar>>
+     */
+    protected function getIndiciesIndexedByIndexName(Table $table): array
```

Comment:
Based on comment above, should this be
```suggestion
    protected function getColumnNamesIndexedByIndexName(Table $table): array
```
?

Reply:
Already done